app/MMB/src/components/alert/__tests__/__snapshots__/Alert.test.js.snap

```diff
@@ -43,12 +43,12 @@ exports[`Alert renders danger type 1`] = `
 <RenderAlert
   containerStyle={
     Object {
-      "backgroundColor": "rgba(208, 2, 27, 0.15)",
```

Comment:
I know it's a snap, but we don't have any transparency anymore, right?

Reply:
Well, when I had the meeting with the orbit designer, he told me we had the wrong colors on our alerts

Reply:
Okay, let's do as the designer says.